common/utils/src/main/resources/error/error-conditions.json

```
@@ -2832,6 +2838,11 @@
           "expects an integer literal, but got <invalidValue>."
         ]
       },
+      "INTERVAL_WITH_MICROSECONDS" : {
+        "message" : [
+          "Cannot add hours, minutes, seconds or microseconds to a date. If necessary set <ansiConfig> to \"false\" to bypass this error."
```

It is not clear from the error message what's wrong. Let's rephrase it:
```suggestion
          "Cannot add an interval because its microseconds part is 0. If necessary set <ansiConfig> to \"false\" to bypass this error."
```